scenes/startup.txt

```diff
@@ -90,6 +92,17 @@
 *create test_stat_charisma 0
 *create test_stat_wisdom 0
 
+*comment TEST DATA for OBJECTS
+*create testobj_fieldname_1 "testfield1"
+*create testobj_fieldname_2 "testfield2"
+*create testobj_fieldname_max 2
+*create testobj_testfield1 ""
+*create testobj_testfield2 ""
+*create testobj_1_testfield1 "Calvin"
```

Comment:
I've found myself getting very confused between testobj_testfield1, testobj_1_testfield1 etc.
Can we rename testfield1 and testfield2 to more realistic examples to help legibility?

"name" and "gold_coins", maybe?

Reply:
Yes, me too :(
I was trying to make it complete generic but I think that was a bad idea. I'll change it.